app/controllers/admin_controller.rb

```diff
@@ -76,7 +76,13 @@ def info
     repository_writable = File.writable?(OpenProject::Configuration.attachments_storage_path)
     @checklist = [
       [:text_default_administrator_account_changed, User.default_admin_account_changed?],
-      [:text_file_repository_writable, repository_writable]
+      [:text_file_repository_writable, repository_writable],
+      [:'extraction.available.pdftotext', TextExtractor::PdfHandler.available?],
+      [:'extraction.available.unrtf',     TextExtractor::RtfHandler.available?],
+      [:'extraction.available.catdoc',    TextExtractor::DocHandler.available?],
+      [:'extraction.available.xls2csv',   TextExtractor::XlsHandler.available?],
+      [:'extraction.available.catppt',    TextExtractor::PptHandler.available?],
+      [:'extraction.available.tesseract', TextExtractor::ImageHandler.available?]
```

Comment:
As we disable the feature completely when 
* running on MySQL 
* not having an enterprise token

it would probably make sense to also note it down here.
We might want to disable the other checks altogether if the precondition (postgresql and enterprise token) is not met.

Reply:
Yes, agreed.

One detail though: If `allows_tsv?` returns `true` we currently do extract attachments even without having a token present. That way, if someone adds a token the feature is up and running straight away without the need of bothering a system adminstrator to start a rake task.

Reply:
👍